src/plugin.rs

```diff
@@ -1,26 +1,17 @@
+use std::any::{type_name, Any};
+
 use crate::AppBuilder;
-// use libloading::{Library, Symbol};
-use std::any::Any;
```

Comment:
Nice rto see we no longer need `Any`. 😄

Reply:
Why? It's not that different than using `std::any::type_name`, right?